packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts

```diff
@@ -214,23 +215,28 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
     buildOptions.scripts,
     'scripts',
   ).reduce((prev: { bundleName: string; paths: string[]; inject: boolean }[], curr) => {
-    const bundleName = curr.bundleName;
-    const resolvedPath = path.resolve(root, curr.input);
+    const { bundleName, inject, input } = curr;
+    const resolvedPath = path.resolve(root, input);
+
+    if (!existsSync(resolvedPath)) {
+      throw new Error(`Script file ${input} doesn't exist.`);
```

Comment:
```suggestion
      throw new Error(`Script file ${input} does not exist.`);
```
nit: avoid contractions

Reply:
Done.

Reply:
Done